rtengine/helpersse2.h

```diff
@@ -117,19 +117,22 @@ static INLINE vfloat vcast_vf_f(float f)
 
 static INLINE vfloat vaddf(vfloat x, vfloat y)
 {
-    return _mm_add_ps(x, y);
```

Comment:
I think some comments why the intrinsics were removed here would be in order. If I remember correctly, you did this due to the compiler not emitting FMA instructions?